index.html

```diff
@@ -473,7 +473,17 @@ <h3>
             "<dfn data-dfn-for="text-direction">auto</dfn>" (default)
           </dt>
           <dd>
-            No explicit directionality.
+            <p>
+              Direction determined from content using the first-strong part of
+              the [[BIDI]] algorithm.
+            </p>
+            <aside class="note">
+              <p>
+                An explicit direction value such as "[=text-direction/rtl=]" or
```

Comment:
Since my above comment makes the meaning of auto less clear, add to the front here: "Using `[=text-direction/auto=]` means that the directionality of each member will be determined by its first strongly directional character. An explicit  ....."